erlang/atomvm/Portfile

```
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+PortGroup       cmake 1.1
+
+github.setup    atomvm atomvm v0.6.0-alpha.0
+categories      erlang devel
+platforms       any
+maintainers     {pguyot @pguyot} openmaintainer
+license         Apache-2
+
+description     Tiny Erlang VM
+long_description    AtomVM implements from scratch a minimal Erlang VM that \
+                    supports a subset of ErlangVM features and that is able to \
+                    run unmodified BEAM binaries on really small systems like \
+                    MCUs.
+                 
+checksums           rmd160  93cc5fbe0dd7096e2577f91a0de4153b36472ee4 \
+                    sha256  18f12adf1230ca4c5d5944d1eb730c4a33232598d2ca6d0bc951e3bd12b32d3e \
+                    size    760390
+
+depends_build       port:erlang \
+                    port:elixir \
+                    port:gperf \
+                    port:cmake
+depends_lib         port:openssl3
```

Same comment as previous:

```suggestion
depends_lib-append  port:openssl3
```